api/overview/azure/latest/communication.networktraversal-readme.md

```diff
@@ -5,10 +5,9 @@ author: acsdevx-msft
 ms.author: acsdevx-msft
 ms.date: 02/09/2022
 ms.topic: reference
-ms.prod: azure
 ms.technology: azure
 ms.devlang: dotnet
-ms.service: communication
+ms.service: azure-communication-services
```

Comment:
This should be updated in https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv